x/emissions/keeper/inference_synthesis/network_inferences.go

```diff
@@ -141,23 +140,36 @@ func GetNetworkInferences(
 	} else {
 		// Single valid inference case
 		singleInference := inferences.Inferences[0]
-
-		networkInferences = &emissions.ValueBundle{
-			TopicId:       topicId,
-			CombinedValue: singleInference.Value,
-			InfererValues: []*emissions.WorkerAttributedValue{
-				{
-					Worker: singleInference.Inferer,
-					Value:  singleInference.Value,
-				},
+		networkInferencesVal := GenerateEmptyValueBundle()
```

Comment:
Making a zero-value struct constructor is much simpler this way:
```go
var x emissions.ValueBundle
x.foo = ...
```
No constructor function needed.

Deeper point-- making a zero-value constructor that passes exhauststruct but then assigns individual values seems to be avoiding the point of exhauststruct.

I think we're going for more semantic clarity.  `yield` @relyt29 right?

Reply:
correct, I actually don't mind the pattern usually but given golang's inflexibility / immaturity of linters to properly handle this case, I think we should force ourselves to just never use this pattern and instead create local variables and then assign them once the full set of all fields is prepared